src/org/traccar/api/resource/UserResource.java

```diff
@@ -50,6 +53,18 @@ public Response add(User entity) throws SQLException {
         if (!Context.getPermissionsManager().isAdmin(getUserId())) {
             Context.getPermissionsManager().checkRegistration(getUserId());
             Context.getPermissionsManager().checkUserUpdate(getUserId(), new User(), entity);
+            Config config = Context.getConfig();
+            int deviceLimit = config.getInteger("default.deviceLimit");
+            int expirationDuration = config.getInteger("default.expirationDuration");
```

Comment:
For this one "users.defaultExpirationDays" would be better to make it more consistent with some other attributes.